### PR TITLE
Added: All the feature of playlist management

### PR DESCRIPTION
**Pull request by hussiantabha**
Now user can make a playlist and add video to it. User can also delete playlist.

### CODE REVIEW

File: src/css/displayvideo.css [hunk 1]
```diff
@@ -49,6 +49,44 @@
...
+.playlist-modal-hide {
+  display: none;
+}
+.playlist-modal {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.5rem;
+  padding: 1rem;
+  width: 280px;
```
Comment (t007rushi) [inline src/css/displayvideo.css]: use rem instead

File: src/css/displayvideo.css [hunk 2]
```diff
@@ -49,6 +49,44 @@
...
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 1;
+  background-color: #e0e7ff;
```
Comment (t007rushi) [inline src/css/displayvideo.css]: use color variable

File: src/pages/DisplayVideo.js [hunk 3]
```diff
@@ -1,103 +1,224 @@
...
+  useEffect(() => {
+    findPlaylist();
+  });
+  useEffect(() => {
+    getVideo();
+  }, []);
+  useEffect(() => {
+    getVideo();
+  }, [param.videoId]);
+  //console.log(videoState);
```
Comment (t007rushi) [inline src/pages/DisplayVideo.js]: remove commenmts

File: src/pages/DisplayVideo.js [hunk 4]
```diff
@@ -1,103 +1,224 @@
...
+            onChange={(e) => setPlaylist(e.target.value)}
+            value={playlist}
+          />
+          <button className="btn btn-primary-outline" onClick={postPlaylist}>
+            Add
+          </button>
+        </div>
+        {videoState.playlist.map((element) => {
+          return (
+            <div className="individual-playlist">
```
Comment (t007rushi) [inline src/pages/DisplayVideo.js]: ```key```?

File: src/pages/DisplayVideo.js [hunk 5]
```diff
@@ -1,103 +1,224 @@
...
+  const getVideo = async () => {
+    const getData = await fetch(`/api/video/${param.videoId}`);
+    const convertedJSON = await getData.json();
     setVideo(convertedJSON.video);
-  });
-  const likeVideo = async (video) => {
-    const postData = await fetch("/api/user/likes", {
+  };
+
+  const findPlaylist = async () => {
```
Comment (t007rushi) [inline src/pages/DisplayVideo.js:23]: can put this outside as services for modularity of code

File: src/pages/Playlist.js [hunk 6]
```diff
@@ -1,7 +1,117 @@
...
+import NormalNavbar from "../components/NormalNavbar";
+import { VideoContext } from "../context/Data";
+import { Link } from "react-router-dom";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 
 const Playlist = () => {
-  return <div>Playlist</div>;
+  const { videoState, dispatch } = useContext(VideoContext);
+  console.log(videoState);
```
Comment (t007rushi) [inline src/pages/Playlist.js:10]: remove the console and why the token is explicitly outside?

File: src/pages/Playlist.js [hunk 7]
```diff
@@ -1,7 +1,117 @@
...
+                      title,
+                      thumbnail,
+                      avatar_url,
+                      creator,
+                      views,
+                      uploaded,
+                      duration,
+                    }) => {
+                      return (
+                        <div className="video-card-container">
```
Comment (t007rushi) [inline src/pages/Playlist.js:81]: same here ```key ={_id}```

**Review by t007rushi — COMMENTED** [review]
look for changes 
else going good 
keep it up

**Review by JantuDeb — APPROVED** [review]

**Pull request merged**